### PR TITLE
Feat/prettier

### PR DESCRIPTION
**Pull request by s-pace**
This PR introduces the use of prettier for scss & md files

solves #515

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
     "docs:serve": "cd ./docs && yarn serve",
     "docs:deploy": "cd ./docs && yarn deploy",
     "lint": "./scripts/lint",
+    "format:scss": "find ./src -type f -name '*.scss' | xargs yarn prettier --write ",
```
Comment (pixelastic) [inline package.json]:
```suggestion
    "format:scss": "prettier --write ./src/**/*.scss",
```
The `./**/*.scss` should grab all `.scss` files in directories and subdirectories.

File: package.json [hunk 2]
```diff
@@ -11,6 +11,8 @@
...
+    "format:md": "yarn prettier --write '**/*.md'",
```
Comment (pixelastic) [inline package.json]:
```suggestion
    "format:md": "prettier --write '**/*.md'",
```
I don't think we need the `yarn` here.

**Comment by s-pace** [comment]
Ready to merge @pixelastic ?

File: package.json [hunk 3]
```diff
@@ -11,6 +11,8 @@
     "docs:serve": "cd ./docs && yarn serve",
     "docs:deploy": "cd ./docs && yarn deploy",
     "lint": "./scripts/lint",
+    "format:scss": "prettier --write ./src/**/*.scss",
+    "format:md": "prettier --write '**/*.md'",
```
[the remark this answers is not in the document]
Comment (s-pace) [reply package.json]: The use of prettier and eslint is not nice with this version (changes are the same with the VS extension but not with the module) so let's continue this way.

**Pull request merged**